mkosi

```diff
@@ -741,7 +741,10 @@ def run_workspace_command(args, workspace, *cmd, network=False, env={}):
                "--register=no",
                "--bind=" + var_tmp(workspace) + ":/var/tmp" ]
 
-    if not network:
+    if network:
+        # If we're using the host network namespace, use the same resolver
+        cmdline += ["--bind-ro=/etc/resolv.conf"]
```

Comment:
Hm, what does this do if `/etc/resolv.conf` is a symlink?

Reply:
Well here the links are followed on both sides and mounted in the appropriate place. 

/dev/vda1 on /run/systemd/resolve/stub-resolv.conf type ext4 (ro,relatime,seclab
el,data=ordered)

Reply:
Hm, looks like it indeed work.

Reply:
I think that same change is missing also for build scipt